pages/common/q.md

```diff
@@ -0,0 +1,23 @@
+# q
+
+> Execute SQL-like queries on .csv and .tsv files.
+
+- Query simple .csv file (with header for column name mapping):
+
+`q -d, -H "SELECT * from {{path/to/file}} where answer = 42"`
```

Comment:
Gotcha. Thanks for the tip.
This is my first open source PR, so I'm still trying to get a better grasp of etiquette.
I would appreciate any other feedback on that front. :)